lib/init/transformations/index.js

```diff
@@ -111,15 +113,16 @@ module.exports = function runTransform(webpackProperties, action) {
 		})
 			.then(_ => {
 				let configurationName;
-				if (!config.configName) {
+				if (!config.configName && action !== "init") {
```

Comment:
You could save the expression `action !== "init"` in variable
```js
const isActionNotInit = action !== "init";
```

Reply:
its used one time, so why bother?

Reply:
No it is not, I see twice in the code changes.